README.md

```diff
@@ -1,13 +1,26 @@
-# gitops-engine
+# GitOps Engine
 
-A place to put:
+Two of the biggest GitOps projects are joining forces. [Argo CD](https://github.com/argoproj/argo-cd) and [Flux CD](https://github.com/fluxcd/flux) have been out there for years, making the lives of operators and organisations easier. Now the teams are building the best possible GitOps solution together. The first step on this journey is the GitOps Engine.
 
-* shared and refactored code from the ArgoCD and CNCF Flux projects
-* specifications and ideas
+Argo and Flux already have similar sets of functionality, soon they will start sharing code using the GitOps Engine. Components of it are going to be e.g.
```

Comment:
Recommend rewording from "Components of it are going to be e.g." to "The GitOps Engine will be responsible for the following functionality:"